brave/src/main/java/brave/internal/recorder/PendingSpans.java

```diff
@@ -140,10 +139,12 @@ void reportOrphanedSpans() {
       );
 
       if (caller != null) {
+        // Verification of this is done manually because log4j2 in our test classpath makes it too
```

Comment:
This comes out of nowhere in business logic and is a little confusing. I'd just remove it - it's common knowledge that verifying log messages is hard and it's very rare to see anyone do it.

Reply:
👍